src/lib/PageHandler.js

```diff
@@ -39,31 +39,24 @@ define([ 'jquery' ], function( $ ) {
     
     this.seekTo = function( target, speed )
     {
-      currentPath !== '' && templateEngine.callbacks[currentPath].exitingPageChange.forEach( function( callback ){
-        callback( currentPath, target );
-      });      
-      
-      var 
-        page = $('#' + target),
-        callbacks = templateEngine.callbacks[target];
-      
+
+      currentPath !== '' && MessageBroker.getInstance().publish("path."+currentPath+".exitingPageChange", currentPath, target);
```

Comment:
See comment in `src/lib/MessageBroker.js` how this line might be simplified to
`currentPath !== '' && MessageBroker.publish("path."+currentPath+".exitingPageChange", currentPath, target);`